db.js

```diff
@@ -490,10 +490,14 @@ exports.init = function (sbot, config) {
     )
     debug(`lowest offset for all indexes is ${lowestOffset}`)
 
+    const B_VALUE = Buffer.from('value')
+
     log.stream({ gt: lowestOffset }).pipe({
       paused: false,
       write(record) {
-        indexesArr.forEach((idx) => idx.onRecord(record, false))
+        const buf = record.value
+        const pValue = bipf.seekKey(buf, 0, B_VALUE)
```

Comment:
We could move these two lines into the internals of `idx.onRecord` (indexes/plugin.js). It's the right place where to do this, and it avoids the duplicated code (lines 512-513 below)

Reply:
But then each plugin would have to calculate them?

Reply:
No, because `onRecord` is a generic wrapper over each plugin's `processRecord` method

Reply:
I still don't understand :-) There is a foreach loop on the next line. Unless we refactor the whole write into a function.